Kate about

Kaka8548 opened the pull request:
Іконка на кнопці "Read more" не выдображається, поки не запушиться гілка Сергія

Kaka8548 committed:
Додано зображення

Kaka8548 committed:
Додано розмітку секції "About" для мобільної версії

Kaka8548 committed:
Merge branch 'main' into kate-about

Kaka8548 committed:
Закоментувано структуру секції

Kaka8548 committed:
Надано стилів секції "About" для моб. версії

Kaka8548 committed:
Виправлено розташування тексту+кнопки

Kaka8548 committed:
Додано стилів секції "About" для таблет-версії

Kaka8548 committed:
Надано стилів секції "About" для десктоп-версії

Kaka8548 committed:
Закоментовано структуру секції (тільки коментарі)

Vlad-Moskalenko reviewed (commented):
https://www.loom.com/share/2faf738f6ccb4050b3d81d33a5d77114

Kaka8548 committed:
Виправлено помилки (далі в описі)

 - ширину екранів змінено на змінні
 - винесено спільний колір у батьківську секцію
 - пофіксено відстані заради піксель-перфект (де це було можливо)
 - видалено зайві класи

Kaka8548 committed:
Виправлено помилку в коді

Kaka8548 commented:
About:

1. У мобільній версії не збігається положення тексту. Це через відсутність леттер спейсінг та лайн хейт у другому параграфі 
2. У десктопній версії не збігається текст з кнопкою, тому що у кожного параграфа в макеті різні розмір тексту, лайн-хейт і леттер-спейсінг. Плюс жовта тінь на макеті криво поїхала відносно картинки, а це впливає на розміри боксу > впливає на центрування тексту відносно картинки. Коротше жах 🥲 Пропоную так залишити. Який макет - такий і код)))

3. no-wrap залишила, бо це мій путь самурая 

4. Перевірила код у валідаторі

Kaka8548 committed:
Змінено кнопку

Kaka8548 committed:
Merge branch 'main' into kate-about
